doc/rule-descriptions.md

```diff
@@ -34,6 +34,7 @@
 | [blink](https://dequeuniversity.com/rules/axe/4.3/blink?application=RuleDescription)                                             | Ensures &lt;blink&gt; elements are not used                                                                   | Serious           | cat.time-and-media, wcag2a, wcag222, section508, section508.22.j                           | failure                    |                                                                                                        |
 | [button-name](https://dequeuniversity.com/rules/axe/4.3/button-name?application=RuleDescription)                                 | Ensures buttons have discernible text                                                                         | Critical          | cat.name-role-value, wcag2a, wcag412, section508, section508.22.a, ACT                     | failure, needs&nbsp;review | [97a4e1](https://act-rules.github.io/rules/97a4e1), [m6b1q3](https://act-rules.github.io/rules/m6b1q3) |
 | [bypass](https://dequeuniversity.com/rules/axe/4.3/bypass?application=RuleDescription)                                           | Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content | Serious           | cat.keyboard, wcag2a, wcag241, section508, section508.22.o                                 | needs&nbsp;review          |                                                                                                        |
+| [color-contrast-enhanced](https://dequeuniversity.com/rules/axe/4.3/color-contrast-enhanced?application=RuleDescription)         | Ensures the contrast between foreground and background colors meets WCAG 2 AAA contrast ratio thresholds      | Serious           | cat.color, wcag2aaa, wcag146                                                               | failure, needs&nbsp;review |                                                                                                        |
```

Comment:
This might've been my fault, due to a botched build and/or commit on my end.  I think I fixed it a minute ago in [this commit](https://github.com/dequelabs/axe-core/pull/3235/commits/7f79fded331fb525417b2a9ed35df8fc04f73b9d).